### PR TITLE
Move all docs content to docs repo, add pointer from README.md

### PR DESCRIPTION
**Pull request by gtardif**
Signed-off-by: Guillaume Tardif <guillaume.tardif@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ See [prerequisites](docs/index.md#prerequisites).
 - [Create a minimal Docker CLI extension](docs/tutorials/minimal-frontend-using-docker-cli.md) - a minimal Desktop Extension containing only a UI part that invokes Docker CLI commands.
 - [Create a ReactJS-based extension](docs/tutorials/react-extension.md) - a minimal Desktop Extension containing only a UI part based on ReactJS.
 
+## Extensions SDK documentation
+
+Documentation about extension SDK and creating your own extensions can be found [here](https://docs.docker.com/desktop/extensions-sdk/).
+
+Pull Requests are welcome to update/improve documentation content at https://github.com/docker/docker.github.io/tree/master/desktop/extensions-sdk
```
Comment (aevesdocker) [inline README.md:24]: Could we change this to keep it inline with the docker.github.io contribution guidelines which asks that updates/improvements are made against the master branch? https://github.com/docker/docker.github.io/blob/master/CONTRIBUTING.md

**Comment by benja-M-1** [comment]
nit: [This](https://github.com/docker/extensions-sdk/blob/main/.gitignore#L2) can be removed as well

File: README.md (hunk 1 above)
Comment (gtardif) [reply README.md:24]: Ah, good, I can point to this CONTRIBUTING page indeed

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Docker Extensions
 
-This repository includes all resources to create Docker Extensions.
+This repository includes extension CLI and samples to create Docker Extensions.
```
Comment (felipecruz91) [inline README.md:3]: Similar to when we use the term "Docker Extensions", I think we should use the term "Extensions CLI". Brand names are typically capitalized. 

@aevesdocker could you confirm? 

```suggestion
This repository includes the Extensions CLI and samples to create Docker Extensions.
```

File: README.md [hunk 3]
```diff
@@ -17,6 +17,12 @@ See [prerequisites](docs/index.md#prerequisites).
 - [Create a minimal Docker CLI extension](docs/tutorials/minimal-frontend-using-docker-cli.md) - a minimal Desktop Extension containing only a UI part that invokes Docker CLI commands.
 - [Create a ReactJS-based extension](docs/tutorials/react-extension.md) - a minimal Desktop Extension containing only a UI part based on ReactJS.
 
+## Extensions SDK documentation
+
+Documentation about extension SDK and creating your own extensions can be found [here](https://docs.docker.com/desktop/extensions-sdk/).
```
Comment (felipecruz91) [inline README.md:22]: same here:

```suggestion
Documentation about the Extensions SDK and creating your own extensions can be found [here](https://docs.docker.com/desktop/extensions-sdk/).
```

**Review by felipecruz91 — APPROVED** [review]
Just minor nits, otherwise LGTM.

File: README.md (hunk 2 above)
Comment (aevesdocker) [reply README.md:3]: Yes agreed @felipecruz91, thanks for catching that

**Pull request merged**